src/WorkOS.net/Services/DirectorySync/Entities/User.cs

```diff
@@ -50,6 +50,12 @@ public class User
         [JsonProperty("emails")]
         public Email[] Emails { get; set; }
 
+        /// <summary>
+        /// The User's groups.
+        /// </summary>
+        [JsonProperty("groups")]
+        public Group[] Groups { get; set; }
```

Comment:
We should actually be using `List`s for collections instead of arrays, as they are the more common collection type in .NET.

While it _should_ be relatively safe to convert the existing properties from arrays to `List`s, it is technically a breaking change. So for now we should just add new properties as `List`s.

```suggestion
        public List<Group> Groups { get; set; }
```